Project/Source/Components/ComponentBoundingBox.cpp

```diff
@@ -92,4 +92,11 @@ const OBB& ComponentBoundingBox::GetWorldOBB() {
 const AABB& ComponentBoundingBox::GetWorldAABB() {
 	CalculateWorldBoundingBox();
 	return worldAABB;
+}
+
+float3 ComponentBoundingBox::GetLocalMinPointAABB() const {
+	return localAABB.minPoint;
+}
+float3 ComponentBoundingBox::GetLocalMaxPointAABB() const {
+	return localAABB.maxPoint;
```

Comment:
These functions are missing a CalculateWorldBoundingBox() call. The localAABB could be outdated.